### PR TITLE
Sensitive review

### PR DESCRIPTION
**Pull request by angolini**
Depends on https://github.com/foundriesio/meta-lmp/pull/652

I tested it using a factory updated to kirkstone, and with DISABLE_GPLV3=1

### CODE REVIEW

File: lmp/bb-config.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+
```
Comment (ricardosalveti) [inline lmp/bb-config.sh:1]: I guess this shouldn't be here?

File: lmp/bb-config.sh [hunk 2]
```diff
@@ -163,6 +163,8 @@ if [ "${DISABLE_GPLV3}" = "1" ]; then
 INHERIT += "image-license-checker lmp-disable-gplv3"
 IMAGE_LICENSE_CHECKER_ROOTFS_BLACKLIST = "GPL-3.0 LGPL-3.0 AGPL-3.0"
 IMAGE_LICENSE_CHECKER_NON_ROOTFS_BLACKLIST = "GPL-3.0 LGPL-3.0 AGPL-3.0"
+IMAGE_LICENSE_CHECKER_ROOTFS_DENYLIST = "GPL-3.0* LGPL-3.0* AGPL-3.0*"
+IMAGE_LICENSE_CHECKER_NON_ROOTFS_DENYLIST = "GPL-3.0* LGPL-3.0* AGPL-3.0*"
```
Comment (quaresmajose) [inline lmp/bb-config.sh:167]: The new variables with `DENYLIST` is added but the old ones `BLACKLIST` is not deleted, shouldn't we erase the old ones?

Comment (angolini) [reply lmp/bb-config.sh:167]: The ci-script at master is used on all factories, so we let the BLACKLIST for backport compatibility (for the factories using v86 and older)

Comment (quaresmajose) [reply lmp/bb-config.sh:167]: Understood, thanks

File: lmp/bb-config.sh (hunk 1 above)
Comment (angolini) [reply lmp/bb-config.sh]: ops

thanks for the review @ricardosalveti

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**